src/databricks/labs/ucx/install.py

```diff
@@ -89,6 +89,8 @@ def deploy_schema(sql_backend: SqlBackend, inventory_schema: str):
     deployer.deploy_view("objects", "queries/views/objects.sql")
     deployer.deploy_view("grant_detail", "queries/views/grant_detail.sql")
     deployer.deploy_view("table_estimates", "queries/views/table_estimates.sql")
+    deployer.deploy_view("misc_patterns", "queries/views/misc_patterns.sql")
```

Comment:
this cannot be done during the main installation phase, which has to run on workspaces without UC enabled. these views rely on `system` schema, that does not exist yet. add a separate temporary CLI command to deploy these views and document it.

Reply:
@nfx Please take a look at the code in these two views. 
They are static SELECT FROM VALUES views to encode easy to change patterns etc.
The `system.access.audit` table is evaluated at open time of the dashboard. The dashboard header advises users of the requirements. Future releases can add commands to ensure system tables are turned on etc.